src/plugins/custom-output-device/index.ts

```diff
@@ -0,0 +1,56 @@
+// import { t } from "@/i18n";
+import promptOptions from "@/providers/prompt-options";
+import { createPlugin } from "@/utils";
+import prompt from "custom-electron-prompt";
+import { onConfigChange, onPlayerApiReady, stop } from "./renderer";
+
+export interface CustomOutputPluginConfig {
+  enabled: boolean;
+  output: string;
+  devices: Record<string, string>;
+}
+
+
+export default createPlugin({
+  name: () => 'Custom Output Device',
+  description: () => 'Configure a custom output media device for songs.',
+  restartNeeded: true,
+  config: {
+    enabled: false,
+    output: 'default',
+    devices: {}
+  } as CustomOutputPluginConfig,
+  menu: async ({ setConfig, getConfig, window }) => {
```

Comment:
You are not using await in this scope.

```suggestion
  menu: ({ setConfig, getConfig, window }) => {
```